src/components/MenuGroup/MenuGroup.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {BrowserRouter as Router} from 'react-router-dom';
+import {render, screen, fireEvent} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import MenuGroup from './index';
+
+describe('MenuGroup component', () => {
+  test('renders without crashing', async () => {
```

Comment:
Do you require `async` here? Try removing it.

Reply:
I’ll fix that, must I’ve been a bad copy paste sorry, 🙏

Reply:
No problem